boms/legacy-ee/pom.xml

```diff
@@ -0,0 +1,5533 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  ~ JBoss, Home of Professional Open Source.
+  ~ Copyright 2010, Red Hat, Inc., and individual contributors
+  ~ as indicated by the @author tags. See the copyright.txt file in the
+  ~ distribution for a full listing of individual contributors.
+  ~
+  ~ This is free software; you can redistribute it and/or modify it
+  ~ under the terms of the GNU Lesser General Public License as
+  ~ published by the Free Software Foundation; either version 2.1 of
+  ~ the License, or (at your option) any later version.
+  ~
+  ~ This software is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+  ~ Lesser General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU Lesser General Public
+  ~ License along with this software; if not, write to the Free
+  ~ Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+  ~ 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.wildfly</groupId>
+        <artifactId>wildfly-parent</artifactId>
+        <!--
+        Maintain separation between the artifact id and the version to help prevent
+        merge conflicts between commits changing the GA and those changing the V.
+        -->
+        <version>27.0.0.Alpha4-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>wildfly-legacy-ee-bom</artifactId>
+    <description>
+		Bill of materials for maven modules that provide traditional capabilities like Jakarta EE
+		and that compile against legacy dependencies		
```

Comment:
By legacy do we mean pre-EE9?

Reply:
Yes. Pre-EE9 and since overridden (as opposed to other stuff that we used pre-EE9 and still use.)

The term is a bit overloaded, as in the source we also use 'legacy' for old subsystems where we now only have admin-only shells for mixed domain.

I have a vague idea that after we clean these boms out by converting the normal source to jakarta.*, the concept may still be useful at some point, in a sense more akin to the mixed domain usage. That is we may still support old versions of stuff (say an EE 10 variant of some spec if we decided to move our standard config to some later variant of that spec.) But we don't need to do that, or we could just drop the 'legacy' boms after we clean them out and then resurrect them later if there is a concrete need.